file_system/models.py

```diff
@@ -545,3 +547,56 @@ def __repr__(self):
 
     def __str__(self):
         return "{}".format(self.machine_name)
+
+
+class PooledNormal(BaseModel):
+    machine = LowercaseCharField(max_length=32, null=False, blank=False)
+    bait_set = LowercaseCharField(max_length=32, null=False, blank=False)
+    gene_panel = models.CharField(max_length=32, null=False, blank=False)
+    preservation_type = models.CharField(max_length=32, null=False, blank=False)
+    run_date = models.DateTimeField()
+    pooled_normals_paths = ArrayField(models.CharField(max_length=255), blank=True, default=list)
+
+    @property
+    def formatted_run_date(self):
+        return self.run_date.strftime("%m-%d-%Y")
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=["machine", "bait_set", "preservation_type"], name="unique_three_fields")
+        ]
+
+    def __repr__(self):
+        return "MACHINE: %s; GENE_PANEL: %s; BAIT_SET: %s" % (self.machine, self.gene_panel, self.bait_set)
+
+    def __str__(self):
+        return "{}".format(self.machine)
+
+    def save(self, *args, **kwargs):
```

Comment:
When are we creating PooledNormal objects? If I understand this correctly we should create PooledNormal objects when we import any new pooledNormals from SMILE?